src/App.js

```diff
@@ -1,8 +1,89 @@
 import React, { Component } from 'react'
+import { css } from 'emotion'
+
+import DropDown from './components/DropDown'
+import SaveButton from './components/SaveButton'
+
+const grid = css`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 5px;
+  grid-template-rows: 50px 70px 70px 50px auto;
+  grid-template-areas:
+    '. goal .'
+    'date date date'
+    'hours hours hours'
+    '. save .'
+    'placeholder placeholder placeholder';
+  width: 240px;
+`
 
 class App extends Component {
+  state = {
+    days: [
+      { date: '2018-06-01', sleeplength: 8, id: 0 },
+      { date: '2018-06-01', sleeplength: 9, id: 1 },
+      { date: '2018-05-31', sleeplength: 7, id: 2 },
+    ],
+    value: 8,
+  }
+
   render() {
-    return <div />
+    return (
+      <div className={grid}>
+        <h2
+          className={css`
+            grid-area: goal;
+          `}
+        >
+          {' '}
+          goal: 8 hours{' '}
+        </h2>
+        <form
+          className={css`
+            grid-area: date;
+          `}
+        >
+          <label> Add night: </label>
+          <input type="date" placeholder="chose date" />
+        </form>
+        <DropDown
+          onChange={e => this.props.handleChange()}
+          onSubmit={e => this.props.handleSubmit()}
```

Comment:
Auch hier die zwei Fehler: du schreibst `this.props`, dabei bekommt doch `<App />` keine props reingereicht. Die Methode ist über `this` direkt zu erreichen. Und 2. musst du auch hier das Event rein reichen, wenn du es in der Methode dann als Parameter erwartest:
```js
onSubmit={e => this.handleSubmit(e)}
```

Reply:
ok